src/node_crypto.cc

```diff
@@ -1076,10 +1076,13 @@ void SecureContext::AddRootCerts(const FunctionCallbackInfo<Value>& args) {
                                            root_cert_store,
                                            extra_root_certs_file.c_str());
       if (err) {
-        ProcessEmitWarning(sc->env(),
-                           "Ignoring extra certs from `%s`, load failed: %s\n",
-                           extra_root_certs_file.c_str(),
-                           ERR_error_string(err, nullptr));
+        if (ProcessEmitWarning(sc->env(),
+                               "Ignoring extra certs from `%s`, "
+                               "load failed: %s\n",
+                               extra_root_certs_file.c_str(),
+                               ERR_error_string(err, nullptr)).IsNothing()) {
+          return;
```

Comment:
This `return` screws up `root_cert_store`'s reference counting

Reply:
Hm – how would this fail? Returning here leaves `root_cert_store` initialized but unused, but the code doesn’t look to me like that’s harmful?

Reply:
`SecureContext::ctx_` won't have the root certificate store associated with it if you return here.  That's a change from before.